app/script/auth/Runtime.js

```diff
@@ -55,15 +57,25 @@ export default class Runtime {
   getBrowserName = () => this.getPlatform().name.toLowerCase();
   getBrowserVersion = () => this.getPlatform().version;
   getMajorBrowserVersion = () => parseInt(this.getPlatform().version.split('.')[0]);
+  getMinorBrowserVersion = () => parseInt(this.getPlatform().version.split('.')[1]);
 
   isSupportedBrowser = () => {
+    const isFranz = this.isElectron() && this.getPlatform().ua.includes('Franz');
+    if (isFranz) {
+      return false;
+    }
+
     return Object.entries(SUPPORTED_BROWSERS).some(([browser, version]) => {
-      return this.getBrowserName() === browser && this.getMajorBrowserVersion() >= version;
+      const isSupportedBrowser = this.getBrowserName() === browser;
+      const isSupportedMajorVersion = this.getMajorBrowserVersion() >= version.major;
+      const isSupportedMinorVersion = this.getMinorBrowserVersion() >= version.minor;
+      return isSupportedBrowser && isSupportedMajorVersion && isSupportedMinorVersion;
```

Comment:
this will fail if the major is bigger while minor is smaller:

version | major | minor
----|----